packages/mui-system/src/cssVars/getInitColorSchemeScript.tsx

```diff
@@ -47,7 +47,7 @@ export interface GetInitColorSchemeScriptOptions {
 
 export default function getInitColorSchemeScript(options?: GetInitColorSchemeScriptOptions) {
   const {
-    enableColorScheme = true,
+    enableColorScheme = false,
```

Comment:
Why was this change here?

Reply:
🤔 It should not change. I will revert it.

Maybe the better name should be `disableColorScheme` but this can be discussed in a separate issue.